extensions/CheckUser/CheckUser.hooks.php

```diff
@@ -189,7 +189,7 @@ protected static function logUserAccountCreation( User $user, $actiontext ) {
 			'cuc_title'      => '',
 			'cuc_minor'      => 0,
 			'cuc_user'       => $user->getId(),
-			'cuc_user_text'  => $user->getName(),
+			// 'cuc_user_text'  => $user->getName(), // SUS-3080 - this column is redundant
 			'cuc_actiontext' => wfMsgForContent( $actiontext ),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)

[project-id]: wikia:app